content/rs/release-notes/rs-6-0-12-january-2021.md

```diff
@@ -129,6 +129,11 @@ To use the updated modules with a database, you must [upgrade the module on the
 - RS45707 - Fixed a bug that caused RCP (Redis Cloud Pro) databases to reject connections while resharding the database.
 - RS51144 - Fixed a bug in the syncer process that was stopping synchronization between all instances in some scenarios of network disconnection of one or more participating clusters.
 
+with 6.0.12-58:
+- RS50865 - Fixed a bug causing rladmin change master node to fail when performed after a prior successful master change.
+- RS51359 - Fixed a memory leak on slave shards in Active-Active databases with replication and AOF for persistence.
```

Comment:
```suggestion
- RS51359 - Fixed a memory leak on replica shards in Active-Active databases with replication and AOF for persistence.
```

Reply:
ohh that's a whole discussion ;) 
I know we should adopt updated terminology, but I think we need to address it when the company officially decides and we have new terminology. Otherwise this can get confusing. 
WDYT?

Reply:
We are trying to replace it where we can, but I'm fine leaving it for now if it'll cause confusion.

Reply:
We know this isn't a simple issue.  For our part, the docs team is updating terminology when used generally.  UX elements are described using the terms displayed on screen.  

We're striving for consistency, clarity, and inclusion throughout the docs.  At least as far as possible.